python/pyADflow.py

```diff
@@ -1682,6 +1692,146 @@ def Func(Xn, CLstar):
                 break
         return Xn
 
+    def solveTargetFuncs(self, aeroProblem, funcDict, tol=1e-4, nIter=10, Jac0=None):
+        """
+        Solve the an arbitrary set of function-dv sets using a Broyden method.
+
+        Parameters
+        ----------
+        AeroProblem : AeroProblem instance
+            The aerodynamic problem to be solved
+        funcDict : dict
+            Dictionary of function DV pairs to solve...add dv index {'func':{'dv':str, 'dvIdx':idx,'target':val,'initVal':val,'initStep':val}}
+        tol : float
+            Tolerance for trimCL solve solution
+        nIter : int
+            Maximum number of iterations.
+        Jac0 : nxn numpy array
+            Initial guess for the func-dv jacobian. Usually obtained
+            from a previous analysis and saves n function
+            evaluations to produce the intial jacobian.
+        """
+
+        self.setAeroProblem(aeroProblem)
+        apDVList = ['alpha','altitude']
```

Comment:
is the apDVList limited to these variables? What about mach, etc.?

Reply:
I can modify this to pull the possible DVs from the aeroproblem